sdk/core/src/index.ts

```diff
@@ -1,49 +1,46 @@
-import {NativeEventEmitter, Platform} from "react-native";
+import { NativeEventEmitter, Platform } from "react-native";
```

Comment:
For all platform-specific APIs add a log for other platform not supported.

Reply:
@umangmoe done